tests/browser/__snapshots__/ssr-hello-world.test.ts.snap

```diff
@@ -1,102 +1,5 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`ssr-hello-world > build > custom port > should generate a production server running on custom port 1`] = `
```

Comment:
This tests a feature of [the server entrypoint](https://github.com/seek-oss/sku/blob/251d4548fd89a0b1d80f1bbc9e2c7e1cc9c7e3c9/packages/sku/src/services/webpack/entry/server/index.ts#L7) itself, separate to the CLI, so IMO worth keeping.